### ISSUE TITLE
please explain

### ISSUE THREAD

**Issue by mike2vandy** [DReichLab/EIG#23]
~/betterSoftware/EIG/bin/smartpca.perl \
     -i samples.eigen.geno \
     -a samples.eigen.snp 
     -b samples.eigen.ind \
     -o test.pca \
     -p test \
     -e test.eigan.eval \
     -l test.log

smartpca -p test.pca.par >test.log
fatalx:
bad chrom: LT635612.1
sh: line 1:  4589 Aborted                 (core dumped) smartpca -p test.pca.par > test.log
ploteig -i test.pca.evec -c 1:2  -p   -x  -y  -o test.xtxt
evec2pca.perl 10 test.pca.evec samples.eigen.ind test.pca

**Comment by bumblenick** [comment]
Alkes Price's lab maintains the perl script but I'll bet that
samples.eigen.snp is
not in reichlab format.  Chromosome values are {1-22, X, Y} and are in the
second column.
snpnames (like LT635612.1 ??) are in the first column.  See examples.

Nick

**Comment by mike2vandy** [comment]
So if my .snp file looks like: 

snp#0   LT635612.1      6e-06   6       G       T
snp#1   LT635612.1      7e-05   70      A       C
snp#2   LT635612.1      9.2e-05 92      T       A
snp#3   LT635612.1      9.3e-05 93      A       T
snp#4   LT635612.1      0.000121        121     C       T
snp#5   LT635612.1      0.000124        124     T       C
snp#6   LT635612.1      0.000125        125     T       C
snp#7   LT635612.1      0.000131        131     A       T
snp#8   LT635612.1      0.000148        148     G       A
snp#9   LT635612.1      0.000159        159     A       T

Can eigensoft not recognize chromosome/contigs names != {1-22, X, Y}? Can that be fixed?

**Comment by bumblenick** [comment]
Nope:  Your C-names won't parse.   You will have to find a way to change
them.

N

**Comment by mike2vandy** [comment]
Imagine if BWA were built this way...

**Comment by bumblenick** [comment]
Feel free to rewrite the software!

Nick
